src/input.test.forms.ts

```diff
@@ -415,12 +433,26 @@ it('is invalid when `required` and `value` does not match `pattern`', async () =
     html`<glide-core-input
       label="Label"
       pattern="[a-z]{4,8}"
+      value="val"
```

Comment:
```suggestion
      value="value"
```

Reply:
Based on the regex and since we want to trigger an **invalid** state, we need `val` here as it's under 4 characters. Could use numbers instead? Or a different regex altogether?

Reply:
Derp. Carry on!